### PR TITLE
Client side boilerplate

### PR DESCRIPTION
**Pull request by justinvdm**
We plan on providing UI for importing content from git.

Since Wagtail's 1.4 release will include [React-based UI components](https://www.npmjs.com/package/wagtail), we plan on also using React for our client-side: for consistency, smaller js bundles, and to be able to extend their components if we need to.

Once importing content from git is in its own molo plugin rather than in core, the plan is for this boilerplate to move there with it.

### CODE REVIEW

**Comment by justinvdm** [comment]
@miltontony ready for review (@codiebeulaine && @saeedmb8 feel free to take a look too if you guys are up for it and have time)

**Comment by miltontony** [comment]
:cake: :tada: 
Is there a way to run the tests in travis

**Comment by justinvdm** [comment]
> Is there a way to run the tests in travis
Yep, will do.

**Comment by justinvdm** [comment]
@miltontony added to travis, ready for re-review.

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "molo",
+  "version": "0.0.0",
+  "description": "An example component demonstrating leet front end techniques we'd like to use for Numi itself.",
```
Comment (miltontony) [inline client/package.json]: :)

**Comment by miltontony** [comment]
:+1: I'm really looking forward to using React

**Pull request merged**